apps/api/src/app/events/e2e/trigger-event-topic.e2e.ts

```diff
@@ -49,7 +49,6 @@ describe('Topic Trigger Event', () => {
 
     beforeEach(async () => {
       process.env.LAUNCH_DARKLY_SDK_KEY = '';
```

Comment:
You can also remove all the process.env.LAUNCH_DARKLY_SDK_KEY references here :) and the afterEach actions

Reply:
Sure, I will remove those as well, good comment